src/solutions/basic_solutions.jl

```diff
@@ -23,13 +23,13 @@ struct LinearSolution{T, N, uType, R, A, C} <: AbstractLinearSolution{T, N}
     u::uType
     resid::R
     alg::A
-    retcode::Symbol
+    retcode::ReturnCode
```

Comment:
This is the only quirk of EnumX: `ReturnCode` is a baremodule, and the type of the enum is called (by default) `T`:
```suggestion
    retcode::ReturnCode.T
```

Reply:
oh interesting, thanks. That makes a lot more sense now.